tests/core/device-reg/task.yaml

```diff
@@ -7,7 +7,7 @@ execute: |
     . "$TESTSLIB"/names.sh
 
     echo "Wait for first boot to be done"
-    while ! snap changes | grep -q "Done.*Initialize system state"; do sleep 1; done
+    retry -n 200 --wait 1 sh -c 'snap changes | MATCH "Done.*Initialize device"'
```

Comment:
Should we use wait_for_first_boot_change here?

Reply:
done